.travis.yml

```diff
@@ -1,24 +1,33 @@
-sudo: required
+sudo: false
+# sudo: enabled
```

Comment:
Question: Should we remove this commented out line, if we already have sudo set to something?